lexicons/app/bsky/notification/putPreferences.json

```
@@ -4,7 +4,7 @@
   "defs": {
     "main": {
       "type": "procedure",
-      "description": "Set notification-related preferences for an account. Requires auth.",
+      "description": "DEPRECIATED: Use putPreferencesV2. Requires auth.",
```

```suggestion
      "description": "DEPRECATED: Use putPreferencesV2. Requires auth.",
```
nit: I think `deprecated` might fit better in this context?

---

Ah shoot I thought it looked wrong. Thanks